SpreadCheetah/SourceGeneration/CellValueConverterAttribute.cs

```diff
@@ -0,0 +1,13 @@
+namespace SpreadCheetah.SourceGeneration;
+
+/// <summary>
+/// Instructs the SpreadCheetah source generator to use specified cell mapper to create a cell.
+/// </summary>
+[AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+public sealed class CellValueConverterAttribute : Attribute
```

Comment:
Maybe we can set the `Type` as a parameter in a primary constructor? E.g.
```csharp
public sealed class CellValueConverterAttribute(Type converterType) : Attribute;
```
It will enforce that a type is actually set, since the attribute can't really be used without it. It will also lead to a bit less verbose code when using the attribute.

For `InheritColumnsAttribute` it made sense to use a property, since that one had an optional parameter.

Reply:
of course 
I forgot about this